x/circuit/features/msg_reset.feature

```diff
@@ -0,0 +1,52 @@
+Feature: MsgResetCircuitBreaker
+
+	- Circuit breaker can be reset:
+	- when the permissions are valid
+
+	Rule: the caller has permission LEVEL_SUPER_ADMIN
+		Example: caller attempts to reset a disabled message
+			Given "acct1" has permission "LEVEL_SUPER_ADMIN"
+			When  attempts to enable a disabled message
+				"""
+				{
+					"msg": "cosmos.bank.v1beta1.MsgSend"
+				}
+				"""
+			Then expect success
```

Comment:
This indent is too deep. How about let's do 2 spaces per indent?

Reply:
weird, tried to undo it, the feature file Is messing with vscode. Ill change it

Reply:
fixed it